src/Component/AOCS/RWModel.cpp

```diff
@@ -93,6 +93,8 @@ void RWModel::MainRoutine(int count) {
   CalcTorque();
 }
 
+void RWModel::PowerOffRoutine() { drive_flag_ = 0; }
```

Comment:
[Q] 再度RWの電源をonするときに，`drive_flag_`は再度1になるようになっていますでしょうか？

Reply:
No, the drive_flag keeps 0. But it is not a problem since the initial value of the drive_flag is 0, and users need to set the flag 1 after turning on the RW. It is the specification of the original RW class and I believe most of the RW components has similar feature (initialize command or activation command).  
If users need automatic flag set feature for a specific component with switching, users can implement the feature by them based on the original RW class.

Reply:
了解です．では，問題ないと思います．